presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraClientConfig.java

```diff
@@ -58,8 +58,6 @@ public void testDefaults()
                 .setDcAwareAllowRemoteDCsForLocal(false)
                 .setUseTokenAware(false)
                 .setTokenAwareShuffleReplicas(false)
-                .setUseWhiteList(false)
```

Comment:
This needs to call `setAllowedAddresses()`

Reply:
✅